projects/ontimize-web-ngx/src/lib/services/ontimize/ontimize-ee.service.ts

```diff
@@ -65,6 +66,10 @@ export class OntimizeEEService extends OntimizeBaseService implements IDataServi
     av = (Util.isDefined(av)) ? av : this.av;
     sqltypes = (Util.isDefined(sqltypes)) ? sqltypes : this.sqltypes;
 
+    kv = NameConvention.parseDataToNameConvention(this._appConfig.nameConvention, kv);
```

Comment:
What happens if an application does not define 'nameConvention' on its appConfig? I do not see any initialization of this parameter

Reply:
It is initializate in [app.config.ts](https://github.com/OntimizeWeb/ontimize-web-ngx/blob/feature/15.x.x-jsonapi-improve-name-convention/projects/ontimize-web-ngx/src/lib/config/app-config.ts), line 115
` getNameConvention() {
    return this._config.nameConvention || 'database';
  }`